src/main/lws/lws-service.js

```diff
@@ -156,14 +164,46 @@ export class LWSService {
 
 	async reqAuth(msg, conn) {
 		try {
+			let check = this.checkWallet(msg.payload.publicKey, conn);
+			if (!check.unlocked) {
+				return conn.send(
+					{
+						error: true,
+						payload: {
+							code: 'auth_error',
+							message: 'Cannot auth with locked wallet'
```

Comment:
are these messages for display in UI as well?

Reply:
Right now - no, but we can make them